app/views/pairings/index.html.slim

```diff
@@ -9,11 +11,17 @@
     thead
       tr
         th Table
+        - if @decks_visible
```

Comment:
this view from your screenshots is confusing because it shows the pairings twice, but with different players listed first.  why this presentation instead of only links on the standings or next to player names in existing pairings views?

Reply:
The first pairings screenshot is the rounds view, the second one is the "Pairings by name" view you get to from the rounds screen. The idea was it lets you see how the decks match up, but it's more useful for spectators or streamers than it is for the players in that pairing.